Ports/tuxracer/patches/0001-Disable-isnan-check.patch

```diff
@@ -0,0 +1,25 @@
+From afa0096774e8e49f91153aaea7cedaa9c9278a77 Mon Sep 17 00:00:00 2001
+From: Grigoris Pavlakis <grigpavl@ece.auth.gr>
+Date: Fri, 27 May 2022 21:25:43 +0300
+Subject: [PATCH 1/4] Disable isnan() check
+
+---
+ configure | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/configure b/configure
+index 20d409b..15d747a 100755
+--- a/configure
++++ b/configure
+@@ -1674,7 +1674,7 @@ $ac_func();
+ 
+ ; return 0; }
+ EOF
+-if { (eval echo configure:1678: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
++if [ "$ac_func" = "isnan" ]; then
```

Comment:
This looks like it will break much more while trying to disable the `isnan` check. Is there a particular reason why we can't just do `export ac_cv_func_isnan=yes` in the `package.sh` script instead of patching `configure`?

Reply:
Not necessarily, it was setup this way so I kept it as it was. Let's try with the export and see what happens